### PR TITLE
extend documentation around chargeback statement billing info

### PR DESCRIPTION
**Pull request by sttomm**
- added payment properties

Concerns CU-6yjzzj

### CODE REVIEW

**Comment by meshcloud-gmbh** [comment]
Task linked: [CU-6yjzzj Extend Chargeback Export to contain more Payment Info](https://app.clickup.com/t/6yjzzj)

File: docs/meshstack.billing.md [hunk 1]
```diff
@@ -239,6 +239,24 @@ The following configuration options are available at `mesh.kraken.productcatalog
 
 Each project in meshStack is associated with a Chargeback Account. meshStack periodically generates [chargeback statements](meshcloud.project-metering.md#chargeback-statements).
 
+The attributes that shall be part of the billing info on the chargeback statements can be configured as relevant meta keys at `mesh.kraken.api.statements`
```
Comment (JohannesRudolph) [inline docs/meshstack.billing.md:242]: f: This does not look like it's sourced from a proper snippet like 
`<!--snippet:mesh.kraken.api.statements-->`

did you write it yourself @sttomm ?

Comment (sttomm) [reply docs/meshstack.billing.md]: yeah, you are right. i manually added it. I now added it as a real snippet

**Comment by sttomm** [comment]
@chewbeccy I extended the config part of this for the operator. can you please check whether the new part in meshstack.billing.md is fine regarding its text?

**Comment by JohannesRudolph** [comment]
@sttomm you'll have to regenerate snippets with the latest develop branch to resolve conflicts (snippets now use dhall code blocks)

**Comment by sttomm** [comment]
i added the note about payment method and tags being taken at the point in time when the chargeback statement is generated. i also updated documentation around payment methods because it was quite deprecated. And i updated the snippet to explain better how to use payment info and tags.

**Review by chewbeccy — APPROVED** [review]

**Pull request merged**